tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/skyrat/face_cursor_combat_mode.tsx

```diff
@@ -0,0 +1,12 @@
+import { multiline } from "common/string";
+import { CheckboxInput, FeatureToggle } from "../../base";
+
+export const face_cursor_combat_mode: FeatureToggle = {
+  name: "Face cursor with combat mode",
+  category: "GAMEPLAY",
+  description: multiline`
+    When toggled, you will now face towards the cursor
+    with combat mode enabled.
+  `,
```

Comment:
What's the advantage of using `multiline` here rather than just using a `\` as you skip the line?

Reply:
I followed examples, this text is too long to be a singleline. I dont know if you can use anything like a `\`